modules/solid_mechanics/doc/content/source/materials/CappedWeakInclinedPlaneStressUpdate.md

```diff
@@ -1 +1,21 @@
-!template load file=stubs/moose_object.md.template name=CappedWeakInclinedPlaneStressUpdate syntax=/Materials/CappedWeakInclinedPlaneStressUpdate
\ No newline at end of file
+# CappedWeakInclinedPlaneStressUpdate
+
+!syntax description /Materials/CappedWeakInclinedPlaneStressUpdate
+
+See [CappedWeakPlaneStressUpdate.md] for details on the
+capped weak plane stress model with a non-inclined weak plane, where
+the weak plane's normal is $(0, 0, 1)$.
+
+This material accepts an arbitrary weak plane normal vector. In the reference frame where
+the inclined plane's normal aligns with the $z$ axis, the assumptions
+listed in [this section](CappedWeakPlaneStressUpdate.md#assumptions) must hold.
+
+See the theory manual  (at [solid_mechanics/doc/theory/capped_weak_plane.pdf](https://github.com/idaholab/moose/modules/solid_mechanics/doc/theory/capped_weak_plane.pdf))
```

Comment:
Does this PDF exist? I thought the `.md` documentation serves as the combined theory and user manuals. The link for this PDF does not work in the compiled version of the docs. I think this can be removed.

Reply:
fixing the link, it does exist